srcpkgs/pass-import/template

```diff
@@ -1,20 +1,13 @@
 # Template file for 'pass-import'
 pkgname=pass-import
-version=3.1
+version=3.2
 revision=1
 build_style=gnu-makefile
-hostmakedepends="python3-setuptools python3-yaml"
-depends="pass>=1.7.0 python3-defusedxml python3-magic
- python3-cryptography python3-SecretStorage python3-pykeepass"
-checkdepends="pass python3-defusedxml python3-magic python3-cryptography
- python3-SecretStorage python3-pykeepass"
+hostmakedepends="python3-setuptools"
+depends="pass python3-defusedxml python3-magic python3-cryptography python3-SecretStorage python3-pykeepass python3-yaml"
 short_desc="Pass extension for importing data from most existing password managers"
 maintainer="Alan Brown <adbrown@rocketmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/roddhjav/pass-import"
 distfiles="https://github.com/roddhjav/pass-import/releases/download/v${version}/pass-import-${version}.tar.gz"
-checksum=659d2b503a18d57c9d63dee216d042d0e24f88576cdd0c7fc4b3426e40dbdbc2
-
-do_check() {
-	python3 setup.py test
-}
+checksum=d943df14dcfec063a6c881c315ce65813db6e105b14a9b0156a73dcf7e529b76
```

Comment:
Why do you remove check?

Reply:
Mainly, because I cannot get `./xbps-src check pass-import` to work on my machine. I have tried with `python3-pytest` and it suggests that there is a problem with `python3-keepass`. I can do some more tests, but as it is optional I removed it.

Reply:
That might be because we have old version of python3-keepass in the repositories, try updating it and see if it helps. Also make sure the tests are ran with pytest, you can then deselect failing tests with `--deselect path/to/failing/test.py`. See for example the [flake8 template](https://github.com/void-linux/void-packages/blob/80b41e8fc25910b0d1daa9c0849a698a3cd516ac/srcpkgs/flake8/template).

Reply:
Have a new pytest check which works. You are right about python3-keepass. I will submit a separate PR to update this, then I will change the test.